eng/pipelines/official/jobs/prepare-signed-artifacts.yml

```diff
@@ -2,6 +2,7 @@ parameters:
   dependsOn: []
   PublishRidAgnosticPackagesFromPlatform: ''
   isOfficialBuild: false
+  logArtifactName: 'Logs-PrepareSignedArtifacts'
```

Comment:
@ViktorHofer where does this parameter get set by callers?

Reply:
The existing caller uses the default value specified.